packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -100,137 +86,114 @@ function ChangelogTooltip<TRef extends HTMLElement>({
   };
 
   return (
-    <BaseTooltip
-      {...props}
-      content={
-        !!post && (
-          <div
-            className="flex relative flex-col whitespace-normal break-words rounded-16 border shadow-2 focus:outline-none w-[24rem] changelog bg-theme-bg-tertiary border-theme-color-cabbage"
-            data-testid="changelog"
+    !!post && (
+      <InteractivePopup
+        {...props}
+        position={InteractivePopupPosition.Unset}
+        className="bottom-0 left-60 mb-6 ml-6 border shadow-2 focus:outline-none w-[24rem] max-w-[360px] max-h-[466px] changelog bg-theme-bg-tertiary border-theme-color-cabbage"
+        data-testid="changelog"
+      >
+        <header className="flex flex-1 items-center py-3 px-4 border-b border-theme-divider-tertiary">
+          <h3
+            className="font-bold typo-title3 text-theme-label-primary"
+            data-testid="changelogNewReleaseTag"
+          >
+            New release
+          </h3>
+          <ModalClose
+            className="right-4"
+            onClick={onModalCloseClick}
+            data-testid="changelogModalClose"
+            buttonSize={ButtonSize.XSmall}
+          />
+        </header>
+        <section className="flex flex-col flex-1 p-5 h-full shrink max-h-full">
+          <Image
+            className="object-cover rounded-lg w-[207px] h-[108px]"
+            alt="Post Cover image"
```

Comment:
```suggestion
            alt="Post cover image"
```
I see it was already a issue :)

Reply:
No worries, can add it 🙌